src/app/core/models/user/user.model.ts

```diff
@@ -18,4 +20,6 @@ export interface User {
   // Business User only
   businessPartnerNo?: string;
   department?: string;
+
+  budgets?: UserBudgets;
```

Comment:
I think, budgets should be held in an individual store within the user management. Do we really need the values outside of the user management?